sim/rogue/rogue.go

```diff
@@ -139,33 +105,35 @@ func (rogue *Rogue) Initialize() {
 	rogue.AutoAttacks.MHConfig().CritMultiplier = rogue.MeleeCritMultiplier(false)
 	rogue.AutoAttacks.OHConfig().CritMultiplier = rogue.MeleeCritMultiplier(false)
 
-	if rogue.Talents.QuickRecovery > 0 {
-		rogue.QuickRecoveryMetrics = rogue.NewEnergyMetrics(core.ActionID{SpellID: 31245})
-	}
-
 	rogue.costModifier = rogue.makeCostModifier()
 
-	rogue.registerStealthAura()
 	rogue.registerBackstabSpell()
 	rogue.registerDeadlyPoisonSpell()
-	rogue.registerPoisonAuras()
 	rogue.registerEviscerate()
 	rogue.registerExposeArmorSpell()
-	rogue.registerFanOfKnives()
 	rogue.registerFeintSpell()
 	rogue.registerGarrote()
 	rogue.registerHemorrhageSpell()
 	rogue.registerInstantPoisonSpell()
 	rogue.registerWoundPoisonSpell()
-	rogue.registerMutilateSpell()
 	rogue.registerRupture()
-	rogue.registerShivSpell()
 	rogue.registerSinisterStrikeSpell()
 	rogue.registerSliceAndDice()
 	rogue.registerThistleTeaCD()
-	rogue.registerTricksOfTheTradeSpell()
 	rogue.registerAmbushSpell()
+
+	// Rune
+	rogue.registerWaylayAura()
+	rogue.registerMasterOfSubtlety()
+	rogue.registerMainGaucheSpell()
+	rogue.registerSaberSlashSpell()
+	rogue.registerShivSpell()
+	rogue.registerShadowstrikeSpell()
+	rogue.registerMutilateSpell()
 	rogue.registerEnvenom()
```

Comment:
I'm not sure if it'll have any impact registering these here vs in the `runes.go` file but the convention we've been following is registering them there

Reply:
Moved rune abilities to match convention